etm/operator-interface-spec/etm-oper-api.yaml

```diff
@@ -675,13 +893,13 @@ components:
         origin:
           type: string
           description: >-
-            CLIENT: Message is from a UTM Client to the UTM System.
-            UTM: Message was automatically generated by the UTM System.
+            CLIENT: Message is from an ETM Client to the ETM System.
+            ETM: Message was automatically generated by the ETM System.
           enum:
             - CLIENT
-            - UTM
```

Comment:
i would leave this because this Project does assert there will be USS-ESS interface